tests/data/ib_traffic_topo_aware_hostfile

```diff
@@ -0,0 +1,471 @@
+ndv414bbc00005I
```

Comment:
is it possible to reduce the scale of test data? current data seems too large
besides, pls make sure the test data does not leak any sensitive information

Reply:
I actually already randomized the physical hostname and the vm hostname in last revision. 
Anyway, I re-faked the test data (hostfile, ibstat, ibnetdiscover,) and its corresponding config file in the latest rev. And it's quite small :)

Reply:
thx, current test data looks great